File: Programming Basics/While Loop - Exercises/07.Moving.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        int width = int.Parse(Console.ReadLine());
        int length = int.Parse(Console.ReadLine());
        int height = int.Parse(Console.ReadLine());

        int freeSpace = width * length * height;

        string boxes = Console.ReadLine();
        int boxesSpace = 0;

        while (boxes != "Done")
        {    
            boxesSpace += int.Parse(boxes);

            if (freeSpace <= boxesSpace)
            {
                Console.WriteLine($"No more free space! You need {Math.Abs(freeSpace - boxesSpace)} Cubic meters more.");
                System.Environment.Exit(0);
            }
            boxes = Console.ReadLine();
        }
        Console.WriteLine($"{freeSpace - boxesSpace} Cubic meters left.");
    }
}

File: Programming Basics - Python/While Loop - Exercises/07.Moving.py
width = int(input())
length = int(input())
height = int(input())

cubic_meters_available = width * length * height

boxes = input()

cubic_meters_used = 0

while boxes != "Done":
    cubic_meters_used += int(boxes)

    if cubic_meters_available < cubic_meters_used:
        print(f"No more free space! You need {cubic_meters_used - cubic_meters_available} Cubic meters more.")
        exit()

    boxes = input()

print(f"{cubic_meters_available - cubic_meters_used} Cubic meters left.")